lib/hooks/update-data/place.js

```diff
@@ -0,0 +1,14 @@
+const { get, set } = require('lodash');
+
+module.exports = (settings, model) => {
+  const isAsru = get(model, 'meta.user.profile.asruUser');
+  const payload = get(model, 'meta.payload');
```

Comment:
This will need to default to `{}` if we don't want it to break when pulling `restrictions` off it later.